hkmc2/shared/src/test/mlscript-compile/Runtime.mls

```diff
@@ -49,7 +49,7 @@ module Tuple with
 
   fun get(xs, i) =
     // * Contrary to `xs.[i]`, this supports negative indices (Python-style)
-    globalThis.Array.prototype.at.call(xs, i)
+    safeCall of globalThis.Array.prototype.at.call(xs, i)
```

Comment:
Check if undefined and throw a proper error, or at least use `checkCall`, not `safeCall`.